iceberg/iceberg-handler/src/test/results/positive/iceberg_truncate_partition_with_evolution.q.out

```diff
@@ -208,6 +208,7 @@ POSTHOOK: Input: default@test_ice_int
 POSTHOOK: Output: hdfs://### HDFS PATH ###
 {"a":11}
 {"a":22}
+{"a":33}
```

Comment:
data changed?

Reply:
Acutualy, this is Apache Iceberg problem, and i have submitted a Apache iceberg issue, see here https://github.com/apache/iceberg/issues/8951 and it has been fixed.
Here I need port the fix(PartitionsTable.java), and we can remove it once a new iceberg version is released.